src/engines_gpl/part/part_data/partfmmem.f90

```diff
@@ -45,12 +45,11 @@ module m_part_flow
     integer :: ndkx              !< dim of 3d flow nodes (internal + boundary)
     integer :: lnkx              !< dim of 3d flow links (internal + boundary)
 
-    integer :: layertype         !< 1= all sigma, 2 = all z, 3 = left sigma, 4 = left z
-
+    integer :: layertype         !< 1 = sigma-layers, 2 = z- or z-sigma-layers, 3 = polygon defined z-layers, 4 = density controlled sigma-layers 
```

Comment:
Here, too, I would move the comments to the respective parameter declarations below.